src/Microsoft.AspNetCore.Razor.Evolution/Legacy/ISymbol.cs

```diff
@@ -5,10 +5,10 @@ namespace Microsoft.AspNetCore.Razor.Evolution.Legacy
 {
     internal interface ISymbol
     {
-        SourceLocation Start { get; }
+        Span Parent { get; set; }
+
         string Content { get; }
 
-        void OffsetStart(SourceLocation documentStart);
-        void ChangeStart(SourceLocation newStart);
+        SourceLocation Start { get; }
```

Comment:
This value is always computed. I'm not sure if we'll want to leave this here long term or not but it was nice for migrating the code.